tests/e2e/lib.sh

```diff
@@ -276,7 +276,16 @@ deploy_sensor_via_operator() {
         --output-secrets - \
     | kubectl -n stackrox apply -f -
 
-    kubectl apply -n stackrox -f tests/e2e/yaml/secured-cluster-cr.yaml
+    template_cr_file=tests/e2e/yaml/secured-cluster-cr-template.yaml
+    gen_cr_file=tests/e2e/yaml/secured-cluster-cr-gen.yaml
+
+
+    # COLLECTION_METHOD should already be set, but let's make 100% sure that it is
+    ci_export COLLECTION_METHOD "${COLLECTION_METHOD:-ebpf}"
```

Comment:
Perhaps this should exit if the COLLECTION_METHOD is not set.

Reply:
I'd like to handle this as:
```
if [[ -n "${COLLECTION_METHOD:-}" ]]; then
  echo "Overriding the product default collection method due to COLLECTION_METHOD variable: ${COLLECTION_METHOD}"
  ... do the template substitution ...
else
  ... do the current apply ...
fi
```
That way we have a means to test the product defaults and to override and set specific values in openshift/release.

This should be removed also in order to have a way to test product defaults:
https://github.com/stackrox/stackrox/blob/8a402a8534f6c4a9d707c6bff9094721fa394387/tests/e2e/lib.sh#L115